specification/eventgrid/data-plane/Microsoft.KeyVault/stable/2018-01-01/KeyVault.json

```diff
@@ -0,0 +1,611 @@
+{
+
+  "swagger": "2.0",
+
+  "info": {
+
+    "version": "2018-01-01",
+
+    "title": "Schema of Azure Vault events published to Azure Event Grid",
+
+    "description": "Describes the schema of the Azure Key Vault events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+
+  },
+
+  "paths": {},
```

Comment:
I cannot see any `paths` defined under this API version. It's quite weird. Is this PR in progress or ready_for_review?

Reply:
I've been guided by EventGrid team (Justin Conway) to add schema for our new events. I believe after that their team is generating API based on that schema.